articles/machine-learning/how-to-machine-learning-interpretability-automl.md

```diff
@@ -118,6 +130,18 @@ engineered_explanations = explainer.explain(['local', 'global'], eval_dataset=au
 print(engineered_explanations.get_feature_importance_dict())
 ```
 
+### Use Mimic Explainer for computing and visualizing raw feature importance
+
+You can call the `explain()` method in MimicWrapper with the transformed test samples to get the feature importance for the generated engineered features. You can also use `ExplanationDashboard` to view the dashboard visualization of the feature importance values of the raw features.
```

Comment:
This sentence is a little confusing because if user is generating **engineered** features, they might want to visualize the engineered feature importances using the `ExplanationDashboard` instead of raw feature importances. We should add a caveat here for what users should expect to see using engineered features vs raw features.

Reply:
Thanks for pointing this out. Fixed it to raw everywhere.